journal.py

```diff
@@ -118,7 +130,26 @@ def view_entry(search_query=None):
 
 def search_entries():
     """Let's us search through the diary entries"""
-    view_entry(input("Enter a search Query: "))
+    while 1:
+        clear()
+        print("What do you want to search for?")
+        print("c) Content")
+        print("t) Tags")
+        print("q) Return to the main menu")
+        print("===============================")
+        print("Action [c/t/q]: ", end="")
```

Comment:
I was adding **python2** support and this line throws me a 

    SyntaxError: invalid syntax

Could you elaborate on what exactly does it do?